src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/Dashboard.test.tsx

```diff
@@ -3278,3 +3279,102 @@ it("send telemetry when users click on exposure chart free scan", async () => {
     }),
   );
 });
+
+it("does not display the CSAT survey banner on the dashboard", () => {
+  const ComposedDashboard = composeStory(
+    DashboardUsPremiumResolvedScanNoBreaches,
+    Meta,
+  );
+  render(<ComposedDashboard />);
+
+  const answerButton = screen.queryByRole("button", {
+    name: "Neutral",
+  });
+  expect(answerButton).not.toBeInTheDocument();
+});
+
+it("displays the CSAT survey banner on the dashboard", () => {
```

Comment:
The current test description seems in direct contradiction to the first test :)

```suggestion
it("displays the CSAT survey banner on the dashboard for Plus users, after more than 90 days since their initial scan", () => {
```

Reply:
Renamed in https://github.com/mozilla/blurts-server/pull/4421/commits/fc50f25631fa956d7f9431e11d8eff8bfb90f2af.